search/src/test/scala/weco/api/search/images/ApiImagesTestBase.scala

```diff
@@ -1,14 +1,19 @@
 package weco.api.search.images
 
 import weco.api.search.ApiTestBase
+import weco.api.search.fixtures.TestDocumentFixtures
+import weco.api.search.json.CatalogueJsonUtil
+import weco.api.search.models.request.SingleImageIncludes
 import weco.catalogue.display_model.test.util.DisplaySerialisationTestBase
 import weco.catalogue.internal_model.generators.ImageGenerators
 import weco.catalogue.internal_model.image._
 
 trait ApiImagesTestBase
     extends ApiTestBase
     with DisplaySerialisationTestBase
-    with ImageGenerators {
+    with ImageGenerators
```

Comment:
Is ImageGenerators still needed? I think you've deleted all the lines that use it.

Reply:
Not in this trait, but it is used in other tests that extend from this one, and removing it will cause conflicts in other PRs – I'm going to leave it for now, and it'll come out when I fully strip out internal model.

(The way we handle traits in tests is a bit messy.)